ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -168,31 +168,22 @@ impl TraceBuilder {
                         let aot_field_off = trace_input_struct_ty.field_byte_off(trace_input_idx);
                         let aot_field_ty = trace_input_struct_ty.field_tyidx(trace_input_idx);
                         // FIXME: we should check at compile-time that this will fit.
-                        match u32::try_from(aot_field_off) {
```

Comment:
Quite hard to see the extent of the change from the diff. Is it just the outermost match that now has an unwrap?

I'd have made the `return Err` a `panic!`, but this is fine too.

Reply:
> Is it just the outermost match that now has an unwrap?

Yes, which matches other cases where we say "ykllvm should/does check this property".